Tasks/DotNetCoreCLI/dotnetcore.ts

```diff
@@ -42,8 +42,9 @@ export class dotNetExe {
                 var output = dotNetExe.getModifiedOutputForProjectFile(this.outputArgument, projectFile);
                 dotnetArguments = this.replaceOutputArgument(output);
             }
-            if(dotnetArguments)
-                dotnet.arg(dotnetArguments.split(" "));
+            if (dotnetArguments) {
```

Comment:
you shouldn't have to check.  tasklib does it for you

https://github.com/Microsoft/vsts-task-lib/blob/master/node/toolrunner.ts#L503